velox/common/memory/Memory.h

```diff
@@ -81,6 +81,12 @@ struct MemoryManagerOptions {
   /// Terminates the process and generates a core file on an allocation failure
   bool coreOnAllocationFailureEnabled{false};
 
+  /// Disables the memory manager's management on memory pools. This also
```

Comment:
I think we just disable memory pool tracking in the memory manager. User can choose to use shared arbitrator or no-op arbitrator. There is no restriction on it. For no-op arbitrator, there is no locking overhead on pool creation. Thanks!